packages/server/src/workers/subscription.test.ts

```diff
@@ -103,6 +103,14 @@ describe('Subscription Worker', () => {
 
     // Update should also trigger the subscription
     expect(queue.add).toHaveBeenCalled();
+
+    // Clear the queue
+    queue.add.mockClear();
+
+    // Delete the patient
+    await repo.deleteResource('Patient', patient.id as string);
```

Comment:
@codyebberson added this line here, as of now the subscription will trigger from every delete resource. I can add to the `matchesCriteria` method an extra conditional that says that `https://medplum.com/fhir/StructureDefinition/subscription-supported-interaction` has to be there && the `valueCode` has to be `delete`